lib/redux/actions/async.js

```diff
@@ -463,6 +470,43 @@ export function clickDeviceSelectionDone() {
   };
 }
 
+export function clickEditUserNext(profile) {
+  return (dispatch, getState) => {
+    const { uploadTargetUser } = getState();
+    const { api } = services;
+    dispatch(syncActions.updateProfileRequest());
```

Comment:
Maybe move this inside the following `if`? After all, we're not making a request if the profile is empty...